src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java

```diff
@@ -30,43 +30,37 @@
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.OneToOne;
+import javax.persistence.Table;
 import javax.persistence.Transient;
 
 import org.osc.sdk.controller.element.NetworkElement;
 
 @Entity
+@Table(name = "NETWORK_ELEMENT")
 public class NetworkElementEntity implements NetworkElement {
 
     @Id
-    @Column(name = "elementId", unique = true)
+    @Column(name = "element_id", unique = true)
     private String elementId;
 
     // TODO : for SFC functionality
     @Transient
     private String parentId;
 
     @ElementCollection(fetch = EAGER)
```

Comment:
why is this eager and portIPs lazy. Should be both eager

Reply:
Last time I tried, that resulted in org.hibernate.loader.MultipleBagFetchException

Reply:
https://stackoverflow.com/questions/37282850/hibernate-org-hibernate-loader-multiplebagfetchexception-cannot-simultaneousl

http://blog.eyallupu.com/2010/06/hibernate-exception-simultaneously.html

Reply:
Tried with sepearate entity for portIps and ManyToOne. Still the same issue.

Reply:
OK. Lazy-loading mac IPs as discussed. But Using FetchType EAGER on both lists and @fetch(SELECT) on one of them also works.